components/ai_chat/core/ai_chat_helper.h

```diff
@@ -0,0 +1,158 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_AI_CHAT_CORE_AI_CHAT_HELPER_H_
+#define BRAVE_COMPONENTS_AI_CHAT_CORE_AI_CHAT_HELPER_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "base/memory/raw_ptr.h"
+#include "base/memory/ref_counted.h"
+#include "base/observer_list.h"
+#include "brave/components/ai_chat/common/mojom/ai_chat.mojom.h"
+#include "brave/components/ai_chat/core/engine/engine_consumer.h"
+#include "components/prefs/pref_change_registrar.h"
+#include "services/data_decoder/public/cpp/data_decoder.h"
```

Comment:
Is this needed?

Reply:
I moved these and refactored the file completely so it is needed now.